extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -162,10 +166,21 @@ export class AmpAnalytics extends AMP.BaseElement {
   layoutCallback() {
     // Now that we are rendered, stop rendering the element to reduce
     // resource consumption.
+    this.element.getResources().setOwner(this.element,
```

Comment:
(Wow, I'm amazed that pasting an image here worked!)
I have not changed any of the mentioned files aside from amp-analytics.js